docs/source/testing/mocking.md

```diff
@@ -77,12 +155,12 @@ server.listen().then(({ url }) => {
 });
 ```
 
-You can also use `mocks` to define object types and the fields belonging to those object types (much like a [resolver map](../data/resolvers/#base-syntax)). 
+You can also use `mocks` to define object types and the fields belonging to those object types (much like a [resolver map](../data/resolvers/#base-syntax)).
 
-In the below example, take note that our `mocks`' `Person` object will invoke a function returning an object with fields pointing at more functions:
+In the below example, note that our mocked `Person` object calls a function returning an object with fields pointing at _other_ functions:
```

Comment:
On second pass I guess we don't typically use "pointing at", and I might recommend something like:

"with fields that contain _other_ functions"